compiler/src/dotty/tools/dotc/core/Contexts.scala

```diff
@@ -490,7 +487,7 @@ object Contexts {
       if releaseName.nonEmpty then ScalaRelease.parse(releaseName).get else ScalaRelease.latest
 
     def tastyVersion: TastyVersion =
-      import math.Ordered.orderingToOrdered
+      //import math.Ordered.orderingToOrdered
```

Comment:
```suggestion
```

Reply:
Dropped the file from this commit to avoid manual conflict with the warning PR.